### ISSUE TITLE
Add voltage references library

### ISSUE THREAD

**Issue by ra3xdh** [ra3xdh/qucs_s#439]
Add library containing SPICE models for voltage references ICs like TL431, AD581, REF191, etc.

**Comment by ra3xdh** [comment]
![image](https://github.com/ra3xdh/qucs_s/assets/4920080/42fed3c1-d571-4007-aa87-16507d2e2202)

**Comment by tomhajjar** [comment]
There are a number of popular National Semi voltage/current references/regulators. LM334, LM385, LM4041, LM4050

Some of these devices can be substitutes for the other. Some come in fixed and adjustable versions. TL431 is more popular than all.

**Comment by ra3xdh** [comment]
Yes, I am planning to add more devices. But I didn't find the model for LM385 yet.

**Comment by tomhajjar** [comment]
It's an older device mostly replaced by the LM4041/4050 series.

**Comment by ra3xdh** [comment]
I have uploaded a new library containing models that I have found. I may add more models if I found something. Closing this as completed.
